Agrume/Agrume.swift

```diff
@@ -217,6 +220,26 @@ public final class Agrume: UIViewController {
     }
     return _spinner!
   }
+	
+	fileprivate var _closeBtn: UIButton?
+	fileprivate var closeBtn: UIButton {
+		if _closeBtn == nil {
+			let closeBtn = UIButton(type: .system)
+			closeBtn.setImage(UIImage(named: "close_btn"), for: UIControlState.normal)
+			closeBtn.tintColor = backgroundColor == UIColor.black ? UIColor.black : UIColor.white
+			closeBtn.addTarget(self, action: #selector(dismissAfterFlick), for: .touchUpInside)
```

Comment:
This should use `#selector(dismiss)` – tapping the button button isn't flicking the image away